model/src/main/java/org/jellyfin/apiclient/model/dlna/DeviceProfile.java

```diff
@@ -517,7 +517,7 @@ public final TranscodingProfile GetVideoTranscodingProfile(String container, Str
 			}
 
 			String tempVar = i.getVideoCodec();
-			if (!StringHelper.equalsIgnoreCase(videoCodec, (tempVar != null) ? tempVar : ""))
+			if(videoCodec != null && !videoCodec.equalsIgnoreCase((tempVar != null) ? tempVar : ""))
```

Comment:
I _think_ we want to `continue` here if `videoCodec` is `null`. Based on how the `equalsIgnoreCase` function worked, `tempVar` would never be null, and if `videoCodec` was null it would return `false` and then be inverted.

Reply:
Yeah that was a mistake on my part trying to fix it quickly. Should be good now!